tests/conftest.py

```diff
@@ -162,3 +166,19 @@ def df():
 @pytest.fixture(scope="session")
 def testdata():
     return TestData()
+
+
+@pytest.fixture
```

Comment:
I feel like we don't really need this. We have [setup_tests](https://github.com/opensearch-project/opensearch-py-ml/blob/main/setup_tests.py) which kind of setup test data in the cluster so that we can perform all the tests. I feel like we can do the same here?

Please take a look at that script and let me know what do you think.

Reply:
I did go through it. Problem is that, setup_tests leave the indexes in the deployment. I was thinking it might be better if our test code take care of both setUp and tear down so that we can test code with existing opensearch clients without leaving test indexes, model data in there. 

I thought of initiating this approach with the test. I was planning to suggest that we delete/restore opensearch to its prior state before tests. What do you think?

Reply:
I am sorry. I got the wrong understanding of what you mentioned.  This `pytest` fixture usage felt like useful as the current one isn't a fixture and we need to manually import in all places.